qa/tasks/scripts/ses_stop_osd_daemon.sh

```diff
@@ -2,11 +2,11 @@ set -ex
 
 declare -a storage_minions=("$@")
 
-random_minion_fqdn=${storage_minions[0]}
-random_minion=${random_minion_fqdn%.*}
-random_osd=$(ceph osd tree | grep -A 1 $random_minion | grep -o osd.* | awk '{print$1}')
+minion_fqdn=${storage_minions[0]}
```

Comment:
The same here, use for to loop through the storage_minions to avoid code duplication.